### ISSUE TITLE
ValueError: 'default' must be a list when 'multiple' is true

### ISSUE THREAD

**Issue by ibrahim-cdprc** [Parallels/rq-dashboard#398]
![image](https://user-images.githubusercontent.com/82905905/131724358-5c5869b9-ecd4-4ca5-8627-4cfeccc0a17b.png)
Not sure what the issue is about

**Comment by alor** [comment]
it's related to: https://github.com/Parallels/rq-dashboard/pull/385

**Comment by arisolt** [comment]
I'm also unable to start rq-dashboard after installing it via pip. Same error as above. Is there a solution to this problem?

**Comment by akcode47** [comment]
Same here

**Comment by huang06** [comment]
workaround: pin the `click` to < 8.0.

```bash
pip install "click>=7,<8"
```

**Comment by ronalddas** [comment]
> workaround: pin the `click` to < 8.0.
> 
> ```shell
> pip install "click>=7,<8"
> ```

After downgrading `click` , am geting new error `ImportError: cannot import name 'ParameterSource' from 'click.core'`

**Comment by 796F** [comment]
same as @ronalddas any luck getting around this?

**Comment by ronalddas** [comment]
> same as @ronalddas any luck getting around this?

Nope, I just used the Job Registries (https://python-rq.org/docs/job_registries/) functions to get whatever data I wanted

**Comment by 796F** [comment]
Gotcha thanks!

**Comment by oodio** [comment]
Having the same issue @796F.

**Comment by peskovkst** [comment]
solved via `pip install git+https://github.com/jace/rq-dashboard.git@main` @oodio

**Comment by ghost** [comment]
This is fixed as of https://github.com/Parallels/rq-dashboard/commit/1be184a83466c84c2188bdbac0fdcd456568cf5e

However this is not yet available on PyPI (#393) so you will need to install from source:
`git+https://github.com/Parallels/rq-dashboard.git@master`.
